src/clib/calc_tdust_1d_g.F

```diff
@@ -344,35 +373,52 @@ subroutine calc_kappa_gr_g(
 !  Arguments
 
       integer in, is, ie
+      integer idspecies
       real*8 t_subl
       real*8 tdust(in)
 
 !  Iteration mask
 
-      logical itmask(in)
+      MASK_TYPE itmask(in)
 
 !  Parameters
 
-      real*8 kgr1, kgr200
-      parameter(kgr1 = 4.0e-4_DKIND, kgr200 = 16.0_DKIND)
+      real*8, parameter :: kgr1 = 4.0e-4_DKIND / 0.009387d0
+      real*8, parameter :: kgr200 = 16.0_DKIND / 0.009387d0
```

Comment:
I concur that this normalization needs to be there. Specifically, equation 17 of Omukai (2000) gives kappa **at Z = Z_local**. However, just below that this paper seems define dust-to-gas ratio at Z_local as 0.934e-2 whereas the value used here is from Pollack (1994). As well, the T^2 proportionality used here is taken from Dopcke et al. (2011), who have a slightly different normalization (by about a factor of 2). This at least needs to be pointed out here.